vendor/vendor.json

```diff
@@ -761,6 +761,12 @@
 			"revision": "fd2f2a919e392b96de74795ae9af2dc5e510bc4c",
 			"revisionTime": "2018-06-06T04:43:31Z"
 		},
+		{
+			"checksumSHA1": "GFsDxJkQz407/2nUBmWuafG+uF8=",
+			"path": "github.com/docker/docker/pkg/term",
+			"revision": "8e908cab46ee73104bedc4cf5fc4a15b7647b500",
+			"revisionTime": "2018-09-25T23:55:46Z"
```

Comment:
I think we should keep the same revision with existing docker package.

Reply:
I have no idea how to keep it. could you give me some suggestions?